src/ui/public/timefilter/timefilter.js

```diff
@@ -23,6 +23,12 @@ uiModules
   var Events = Private(EventsProvider);
   var diff = Private(UtilsDiffTimePickerValsProvider);
 
+  config.$bind($rootScope, 'dateFormat:dow', 'dateFormat_dow');
+
+  $rootScope.$watch('dateFormat_dow', function (day) {
```

Comment:
Why not put it in the timpicker directive file, in the controller? It appears to me that people have to use the directive in order to use this.